templates/snippets/footer.html

```diff
@@ -21,17 +21,23 @@
                 <img src="{% static 'img/logos/cdh-logo.svg' %}" alt="The Center for Digital Humanities at Princeton" />
             </a>
         </div>
-        <section class="social">
-            <a href="https://www.facebook.com/ProsodyArchive/">
-                <img src="{% static 'img/icons/facebook.svg' %}" alt="ProsodyArchive on Facebook"/>
-            </a>
-            <a href="https://twitter.com/ProsodyArchive">
-                <img src="{% static 'img/icons/twitter.svg' %}" alt="ProsodyArchive on Twitter"/>
-            </a>
-            <a href="https://www.instagram.com/prosodyarchive/">
-                <img src="{% static 'img/icons/instagram.svg' %}" alt="ProsodyArchive on  Instagram"/>
-            </a>
-        </section>
+        <div class="media">
+            <div class="left">
+                <a class="license" href="https://creativecommons.org/licenses/by/4.0/"><img src="{% static 'img/logos/cc-by.svg' %}"></a>
```

Comment:
shouldn't we have some alt text for a screen-readable label saying what the license is? (creative commons attribution?)

Reply:
oh, duh, thank you! Will add